packages/lit-dev-tools-cjs/src/search/plugin.ts

```diff
@@ -55,26 +60,37 @@ export async function createSearchIndex(outputDir: '_dev' | '_site') {
     `../../../lit-dev-content/${outputDir}/searchIndex.json`
   );
   const docs: UserFacingPageData[] = await indexDocs(outputDir);
-  let idOffset = Number(docs[docs.length - 1].objectID);
+  let idOffset = Number(docs[docs.length - 1]?.objectID);
```

Comment:
Does optional chaining risk passing `undefined` as an idOffset? Worth explicitly checking or using `!` instead so we get a crash if something goes wrong?

Reply:
great suggestion, I like that approach